web/src/app/administration/components/workbasket-access-items/workbasket-access-items.component.spec.ts

```diff
@@ -91,44 +102,10 @@ describe('WorkbasketAccessItemsComponent', () => {
   });
 
   afterEach(() => {
-    document.body.removeChild(debugElement);
+    fixture.destroy();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
-
-  it('should show two access items if server returns two entries', () => {
-    expect(debugElement.querySelectorAll('#table-access-items > tbody > tr').length).toBe(2);
-  });
-
-  it('should remove an access item if remove button is clicked', () => {
-    expect(debugElement.querySelectorAll('#table-access-items > tbody > tr').length).toBe(2);
-    debugElement.querySelectorAll('#table-access-items > tbody > tr')[0].querySelector('td > button').click();
-    fixture.detectChanges();
-    expect(debugElement.querySelectorAll('#table-access-items > tbody > tr').length).toBe(1);
-  });
-
-  it('should show success alert after saving', async(() => {
```

Comment:
We do we not continue to test these tests? 
Are they broken or unneccessary?
If broken, why not fix them or create a ticket for fixing them, while ignoring them (with xit)

Reply:
we can ignore them and overhaul the whole unit tests